tests/integration/test_fim/test_registry/test_registry_wildcards/test_registry_wildcards.py

```diff
@@ -0,0 +1,249 @@
+    - Windows Server 2003
+    - Windows XP
+
+references:
+    - https://documentation.wazuh.com/current/user-manual/capabilities/file-integrity/index.html
+    - https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/syscheck.html#file-limit
+
+pytest_args:
+    - fim_mode:
+        scheduled: file/registry changes are monitored only at the configured interval
+    - tier:
+        0: Only level 0 tests are performed, they check basic functionalities and are quick to perform.
+        1: Only level 1 tests are performed, they check functionalities of medium complexity.
+        2: Only level 2 tests are performed, they check advanced functionalities and are slow to perform.
+
+tags:
+    - fim_registry_wildcards
+'''
+import os
+import time
+import pytest
+from wazuh_testing import LOG_FILE_PATH, T_10
+from wazuh_testing.tools.configuration import load_configuration_template, get_test_cases_data
+from wazuh_testing.tools.monitoring import FileMonitor, generate_monitoring_callback
+from wazuh_testing.modules import WINDOWS, TIER1
+from wazuh_testing.modules.fim import (registry_parser, KEY_WOW64_64KEY, REG_SZ,
+                                       WINDOWS_HKEY_LOCAL_MACHINE)
+from wazuh_testing.modules.fim import FIM_DEFAULT_LOCAL_INTERNAL_OPTIONS as local_internal_options
+from wazuh_testing.modules.fim.event_monitor import (CB_FIM_WILDCARD_EXPANDING, callback_key_event, get_messages,
+                                                     check_registry_crud_event, callback_value_event)
+from wazuh_testing.modules.fim.utils import (create_registry, modify_registry_value, delete_registry,
+                                             delete_registry_value)
+
+# Marks
+pytestmark = [WINDOWS, TIER1]
+
+# Reference paths
+TEST_DATA_PATH = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+CONFIGURATIONS_PATH = os.path.join(TEST_DATA_PATH, 'configuration_template')
+TEST_CASES_PATH = os.path.join(TEST_DATA_PATH, 'test_cases')
+
+# Configuration and cases data
+configurations_path = os.path.join(CONFIGURATIONS_PATH, 'configuration_registry_wildcards.yaml')
+t1_cases_path = os.path.join(TEST_CASES_PATH, 'cases_registry_key_wildcards.yaml')
+t2_cases_path = os.path.join(TEST_CASES_PATH, 'cases_registry_value_wildcards.yaml')
+
+# Enabled test configurations (t1)
+t1_configuration_parameters, t1_configuration_metadata, t1_case_ids = get_test_cases_data(t1_cases_path)
+t1_configurations = load_configuration_template(configurations_path, t1_configuration_parameters,
+                                                t1_configuration_metadata)
+
+t2_configuration_parameters, t2_configuration_metadata, t2_case_ids = get_test_cases_data(t2_cases_path)
+t2_configurations = load_configuration_template(configurations_path, t2_configuration_parameters,
+                                                t2_configuration_metadata)
+
+
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+key_name = 'test_key'
+value_name = 'test_value'
```

Comment:
Add a comment that identifies this section.

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/3856/commits/b50ad9625566811faf8e88847b0fb2542f4bcb69